### PR TITLE
30k30days - student interview videos

### PR DESCRIPTION
**Pull request by juemura**
I know @etrepum added the playlist, but Stevon wanted the videos on the page (4 videos in a row). Is it confusing if we have both?

Below are the two options: 2x2 and 1x4.
mission bit 30k in 30 days 2x2
mission bit 30k in 30 days 1x4

### CODE REVIEW

File: 30k30days/index.html [hunk 1]
```diff
@@ -90,6 +91,36 @@ <h3><strong class="format-money">${{ total }}</strong> Raised of $30,000 Goal ({
         We give students the opportunity to learn directly from successful engineers and entrepreneurs who enjoy sharing knowledge. We're focused on project based learning and aim to teach both core computer science concepts as well as the practical skills and technologies that are required to build web sites and applications today. We currently run classes in San Francisco and Oakland.
       </p>
     </div>
+
+    <div class="row text-center">
+      <h2>Meet our Students</h2>
+      <div class="col-md-3">
+        <h3>Abel Regalado</h3>
+        <div class="video-responsive">
+          <iframe width="560" height="315" src="https://www.youtube.com/embed/33L0e_u1I5s" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
```
Comment (etrepum) [inline 30k30days/index.html]: nit: we should add `?rel=0&modestbranding=1`  to these so we don't get related videos or as much youtube branding

**Comment by etrepum** [comment]
I think that the 2x2 layout is a bit more aesthetically pleasing.

I wonder if "Meet our Students" might make people think that Mission Bit only has four students? :) I don't have any immediate ideas for different copy there, but something to think about.

File: 30k30days/index.html (hunk 1 above)
Comment (juemura) [reply 30k30days/index.html]: good catch.

**Comment by etrepum** [comment]
Maybe we say Featured Students?

This is what it looks like after using col-md-6 with that copy:

image

**Comment by etrepum** [comment]
I think we should probably leave the main content to be the playlist, I think people will figure it out if they let it play long enough to see some of the student profiles

**Comment by juemura** [comment]
I like "meet our students" better. I don't think it suggests we only have 4, specially given that there are photos of a bunch of different students all over the page and the rest of the website.
I changed the page to only show first names because one of the kids' parent doesn't want the last name featured, so just for consistency. I'll change that on youtube as well.

**Comment by etrepum** [comment]
What do you think about going to the 2x2 col-md-6 layout?

**Comment by juemura** [comment]
I like the 2x2 as well, but folks here at the office all voted for the 1x4.
Why does the font look different in your screenshot?

**Comment by etrepum** [comment]
The benefit of 2x2 is that it's much bigger than the social media feed below it, so they don't get visually lost in that noise, and that you can watch the video as-is on the page without going full screen. At 4x1 it's like watching a postage stamp.

**Comment by juemura** [comment]
I agree! Let's just make it 2x2.

**Comment by etrepum** [comment]
I don't know why exactly the font is showing differently, but this is Windows. Looks like Arial, since Helvetica Neue and Helvetica aren't installed by default.

**Pull request merged**